robottelo/cli/docker.py

```diff
@@ -69,7 +69,7 @@ def create(cls, options=None):
                                                       values.
             --registry-id REGISTRY_ID                 Registry this container
                                                       will have to use to get
-                                                      the image
+                                                      the manifest
```

Comment:
I am actually not see that information for latest SNAP, please update code correspondingly
```
>hammer docker container create --help
...
 --organizations ORGANIZATION_NAMES        Comma separated list of values.
 --registry REGISTRY_NAME                  Name to search by
 --registry-id REGISTRY_ID                  
 --repository-name REPOSITORY_NAME         Name of the repository to use to create the container. e.g. centos
 --tag TAG                                 Tag to use to create the container. e.g. latest
...
```